google/cloud/automl/unit-tests/cloud_automl_vision_model_tests.yaml

```diff
@@ -0,0 +1,86 @@
+test:
+  suites:
+  - name: Cloud AutoML Vision Model API IT
+    setup:
+    - code: cloud_region = "us-central1"
+      # Read Project ID from Environment Variable
+    - env:
+        what: GOOGLE_PROJECT_ID
+        var: project_id
+    - env:
+        what: MODEL_ID
```

Comment:
```yaml
- env:
    name: MODEL_ID
    variable: model_id
```

If your sample-tester uses `what`, you should update to use the latest sample-tester

```
pip install sample-tester
```